modules/database/sqlite/site-health.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Where is this file being `require_once`d? That may be missing from the module's `load.php` file.

Reply:
🤦 you're right. Added it 👍